web-app/src/main/webapp/images/map.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="41" height="37" viewBox="0 0 41 37" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
if these came from elsewhere, make sure you have licenses for them

Reply:
No attribution necessary according to the [github](https://google.github.io/material-design-icons/#licensing)